app/dubbo-cp/cmd/run.go

```diff
@@ -122,10 +121,10 @@ func newRunCmdWithOpts(opts dubbo_cmd.RunCmdOpts) *cobra.Command {
 				runLog.Error(err, "unable to set up DP Server")
 				return err
 			}
-			if err := defaults.Setup(rt); err != nil {
-				runLog.Error(err, "unable to set up Defaults")
-				return err
-			}
+			//if err := defaults.Setup(rt); err != nil {
+			//	runLog.Error(err, "unable to set up Defaults")
+			//	return err
+			//}
```

Comment:
Sorry, because in universal mode, the program stops after ten minutes of startup, which is caused by the failure of this component in the program to start successfully. Therefore, when I tested the functionality, I commented it out, but I forgot to restore it when I submitted it.